packages/tunnel/src/commands/deploy/index.ts

```diff
@@ -0,0 +1,109 @@
+import { deployToLogtoCloud } from './utils.js';
+
+const tunnel: CommandModule<unknown, DeployCommandArgs> = {
+  command: ['deploy'],
+  describe: 'Deploy your custom UI assets to Logto Cloud',
+  builder: (yargs) =>
+    yargs
+      .options({
+        auth: {
+          describe:
+            'Auth credentials of your Logto M2M application. E.g.: <app-id>:<app-secret> (Docs: https://docs.logto.io/docs/recipes/interact-with-management-api/#create-an-m2m-app)',
+          type: 'string',
+        },
+        endpoint: {
+          describe:
+            'Logto endpoint URI that points to your Logto Cloud instance. E.g.: https://<tenant-id>.logto.app/',
+          type: 'string',
+        },
+        path: {
+          alias: ['experience-path'],
+          describe: 'The local folder path of your custom sign-in experience assets.',
+          type: 'string',
+        },
+        resource: {
+          alias: ['management-api-resource'],
+          describe: 'Logto Management API resource indicator. Required if using custom domain.',
+          type: 'string',
+        },
+        verbose: {
+          describe: 'Show verbose output.',
+          type: 'boolean',
+          default: false,
+        },
+      })
+      .epilog(
+        `Refer to our documentation for more details:\n${chalk.blue(
+          'https://docs.logto.io/docs/references/tunnel-cli/deploy'
+        )}`
+      ),
+  handler: async (options) => {
+    const {
+      auth,
+      endpoint,
+      path: experiencePath,
+      resource: managementApiResource,
+      verbose,
+    } = options;
+    if (!auth) {
+      consoleLog.fatal(
+        'Must provide valid Machine-to-Machine (M2M) authentication credentials. E.g. `--auth <app-id>:<app-secret>` or add `LOGTO_AUTH` to your environment variables.'
+      );
+    }
+    if (!endpoint || !isValidUrl(endpoint)) {
+      consoleLog.fatal(
+        'A valid Logto endpoint URI must be provided. E.g. `--endpoint https://<tenant-id>.logto.app/` or add `LOGTO_ENDPOINT` to your environment variables.'
+      );
+    }
+    if (!experiencePath) {
+      consoleLog.fatal(
+        'A valid experience path must be provided. E.g. `--experience-path /path/to/experience` or add `LOGTO_EXPERIENCE_PATH` to your environment variables.'
+      );
+    }
+    if (!existsSync(path.join(experiencePath, 'index.html'))) {
+      consoleLog.fatal(`The provided experience path must contain an "index.html" file.`);
+    }
+
+    const spinner = ora();
+
+    if (verbose) {
+      consoleLog.info(
+        `${chalk.bold('Starting deployment...')} ${chalk.gray('(with verbose output)')}`
+      );
+    } else {
+      spinner.start('Deploying your custom UI assets to Logto Cloud...');
+    }
+
+    await deployToLogtoCloud({ auth, endpoint, experiencePath, managementApiResource, verbose });
+
+    if (!verbose) {
+      spinner.succeed('Deploying your custom UI assets to Logto Cloud... Done.');
+    }
+
+    await sleep(1000);
```

Comment:
is it necessary?

Reply:
Personally this 1s timeout feels better. Will clearly see the success message of the first one as the 1st step, and then flood the screen with more information.

Reply:
Well, removed